src/components/App.js

```diff
@@ -0,0 +1,40 @@
+import Navbar from './Navbar'
+import CountriesList from './ContriesList'
+import CountriesDetails from './CountryDetails'
+import './App.css'
+
+import React, { Component } from 'react'
+import { Switch, Route, } from 'react-router-dom'
+import axios from 'axios'
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      countries:[]
+    }
+  }
+
+  componentDidMount = () => {
+    axios
+      .get('https://countries.tech-savvy.tech/countries')
+      .then(res => this.setState({ countries: res.data }))
+  }
+
+  render() {
+    return(
+    <>
+      <Navbar title='WikiCountries' />
+      
+      <CountriesList countries={this.state.countries} />
+      <main class='main'>
```

Comment:
```suggestion
      <main className='main'>
```
Recuerda estar revisando los warning en consola